chips/sam4l/src/adc.rs

```diff
@@ -845,6 +846,13 @@ impl hil::adc::Adc for Adc {
     fn get_voltage_reference_mv(&self) -> Option<usize> {
         Some(3300)
     }
+
+    /// Sets the client for this driver.
+    ///
+    /// - `client`: reference to capsule which handles responses
+    fn set_client(&self, _client: &'static dyn hil::adc::Client) {
+        panic!("This is not implemented yet!");
```

Comment:
There is an `unimplemented` macro you can use instead

Reply:
There is already a set_client function which should be updated to this one.

Reply:
The existing set_client function is not in the HIL and is defined for EverythingClient. Can you be a little bit more specific? I am not sure I understand the request.